code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -92,8 +101,8 @@
 	///An "override" color that can be applied to ANY limb, greyscale or not.
 	var/variable_color = ""
 
-	///whether it can be dismembered with a weapon.
-	var/dismemberable = 1
+	///whether the limb can be mutilated, including dismemberment for appendages and heads, and disembowelment for chests. TRUE means it can be subjected to these effects.
+	var/mutilation_allowed = TRUE
```

Comment:
worse var nake imo. dismemberable affects whether or not dismember() does anything.

Reply:
Well, my reasoning is that dismember() is somewhat deceptive. In the case of chests, rather than dismembering, it disembowels. I suppose since I'm not changing that proc name, I probably shouldn't change this, you're right.

3am brain logic.